src/components/01-branding/02-typography/paragraph/paragraph-default.config.yml

```diff
@@ -0,0 +1,13 @@
+label: Paragraph
```

Comment:
paragraph-default.config.yml filename should just be paragraph.config.yml. After the change having the label set for the base or any of the variants will be unnecessary